README.md

```diff
@@ -1,31 +1,133 @@
-micropython-lib
-===============
+# micropython-lib
 
-This is a repository of libraries designed to be useful for writing
-MicroPython applications.
+This is a library of packages designed to be useful for writing MicroPython
+applications.
 
-The libraries here fall into four categories corresponding to the four top-level directories:
+The packages here fall into four categories corresponding to the four
```

Comment:
No need to duplicate 'four'. "The packages here fall into categories corresponding to the four top-level directories" or "The packages here fall into four categories corresponding to the top-level directories"